src/azul/indexer/index_service.py

```diff
@@ -442,9 +443,13 @@ def _aggregate(self, contributions: List[CataloguedContribution]) -> List[Aggreg
                      version=c.coordinates.bundle.version)
                 for c in contributions
             ]
+            source = one(set(c.source for c in contributions))
             aggregate_cls = self.aggregate_class(entity.catalog)
             aggregate = aggregate_cls(coordinates=AggregateCoordinates(entity=entity),
                                       version=None,
+                                      # FIXME: There's a type warning here and I
```

Comment:
If the warning exists on develop, don't worry about it. Otherwise, you gotta figure it out.